docs/ops/movement/Slice_8.md

```diff
@@ -0,0 +1,447 @@
+## Slice <a name="Slice"></a> {#openvino_docs_ops_movement_Slice_8}
+
+**Versioned name**: *Slice-8*
+
+**Category**: *Data movement*
+
+**Short description**: *Slice* operation extracts a slice of the input tensor.
+
+**Detailed Description**: *Slice* operation selects a region of values from the `data` tensor.
+Selected values start at indexes provided in the `start` input and end
+at indexes provides in `stop` input (exclusively).
+
+Input `step` allows subsampling of `data`, selecting evey *n*-th element,
+where `n` is equal to `step` element for corresponding axis.
+Negative `step` value indicates slicing backwards, so the sequence along the corresponding axis is reversed in the output tensor.
+
+Optional `axes` input allows specifying slice indexes only on selected axes.
+Other axes will not be affected and will be output in full.
+
+The rules follow python language slicing `data[start:stop:step]`.
+
+**Attributes**: *Slice* operation has no attributes.
+
+**Inputs**
+
+* **1**: `data` - tensor (to be sliced) of type *T* and shape rank greater or equal to 1. **Required.**
+
+* **2**: `start` - 1D tensor of type *T_IND*. Indices corresponding to axes in `data`.
+  Defines the starting coordinate of the slice in the `data` tensor.
+  A negative index value represents counting elements from the end of that dimension.
+  A value larger than the size of a dimension is silently clamped. **Required.**
```

Comment:
should we tell that dimension size of `start` (`stop` and `step`) must be equal to a rank of `data`?

Reply:
The size of these inputs can be smaller than rank of the `data` input, if not all of the axes are specified. 
This is mentioned in the input `axes` and detailed description, at the and there is additional info:
"Number of elements in `start`, `stop`, `step`, and `axes` inputs are required to be equal."